app/models/planning_application.rb

```diff
@@ -25,7 +25,7 @@ class WithdrawOrCancelError < RuntimeError; end
 
   with_options dependent: :destroy do
     has_many :audits, -> { by_created_at }, inverse_of: :planning_application
-    has_many :documents, -> { with_file_attachment.by_created_at }, inverse_of: :planning_application
```

Comment:
Ah did this default scope not work when calling the association?

Reply:
It was doing some good things in some places, and in some places was loading it unnecessarily

Reply:
Yeah makes sense